openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMBinaryClassificationModelTrainer.java

```diff
@@ -58,43 +54,64 @@
      */
     private static final Logger logger = LoggerFactory.getLogger(LightGBMBinaryClassificationModelTrainer.class);
 
-
     /**
      * This class is not meant to be instantiated.
      */
     private LightGBMBinaryClassificationModelTrainer() {}
 
     /**
-     * Train a LightGBM model from scratch, using train data from an in-memory dataset.
+     * Train a LightGBM model from scratch, using streamed train data from a dataset iterator.
```

Comment:
Javadocs render html, not markdown (e.g., replace `**Problem**` by `<b>Problem</b>`